routes/index.js

```diff
@@ -1,8 +1,9 @@
 const express = require("express");
 const router = express.Router();
 
-const userRouter = require("./userRouter.js");
+const userRouter = require("./userRouter");
+const orderRouter = require("./orderRouter");
 
 router.use("/users", userRouter.router);
-
+router.use("/orders", orderRouter.router);
```

Comment:
아래 코드와 공백을 두어 코드 구분을 해주세요!

가독성을 향상시키고 유지보수도 용이해집니다.
```suggestion
router.use("/orders", orderRouter.router);

```